ksql-engine/src/main/java/io/confluent/ksql/planner/plan/AggregateNode.java

```diff
@@ -364,13 +364,12 @@ private Schema buildAggregateSchema(final Schema schema,
     }
     for (int aggFunctionVarSuffix = 0;
          aggFunctionVarSuffix < getFunctionList().size(); aggFunctionVarSuffix++) {
-      Schema fieldSchema;
       String udafName = getFunctionList().get(aggFunctionVarSuffix).getName()
           .getSuffix();
       KsqlAggregateFunction aggregateFunction = functionRegistry.getAggregateFunction(udafName,
           getFunctionList()
               .get(aggFunctionVarSuffix).getArguments(), schema);
-      fieldSchema = aggregateFunction.getReturnType();
+      Schema fieldSchema = aggregateFunction.getReturnType();
```

Comment:
Can probably take this one step further and just inline it. There is no reason for having the variable